Examples/TicTacToe/tic-tac-toe/Tests/GameSwiftUITests/GameSwiftUITests.swift

```diff
@@ -5,83 +5,62 @@ import XCTest
 @testable import GameSwiftUI
 
 class GameSwiftUITests: XCTestCase {
```

Comment:
These tests became a lil less exhaustive, but I tried to:

1. Always exhaustively test against state after the first action.
2. Avoid re-testing similar actions in variant tests.